api/helpers/validation/customerValidation.js

```diff
@@ -81,6 +81,12 @@ class CustomerValidation {
             throw new UpdateError('One or more updated fields are empty.');
         }
     }
+
+    static ValidateIdExists(id) {
+        if (id === null || id === undefined) {
+            throw new Error();
```

Comment:
what about this error?

Reply:
Frankly I left it as just an "Error" because the message which we have getting that error is from Domain, so in this case that's IdNotFound. So that's working fine anyway. So should I change that Error above to IdNotFound anyway?